state/futures/reducer.ts

```diff
@@ -168,6 +171,7 @@ export const FUTURES_INITIAL_STATE: FuturesState = {
 		leverageInput: '0',
 	},
 	tradePanelDrawerOpen: false,
+	marketFundingRates: [],
```

Comment:
Would be nice to store funding rates by market so the data remains cached for speedy loading when switching around.

Also we'll probably want to bump the store version.

Reply:
Pushed.